src/transformers/models/gemma3/modeling_gemma3.py

```diff
@@ -1270,20 +1270,21 @@ def forward(
         >>> import requests
         >>> from transformers import AutoProcessor, Gemma3ForConditionalGeneration
 
-        >>> model = Gemma3ForConditionalGeneration.from_pretrained("google/Gemma3-test-224px-hf")
-        >>> processor = AutoProcessor.from_pretrained("google/Gemma3-test-224px-hf")
+        >>> model = Gemma3ForConditionalGeneration.from_pretrained("google/gemma-3-4b-it")
```

Comment:
This model also isn't available in the docstring [example](https://huggingface.co/docs/transformers/main/en/model_doc/gemma3#transformers.Gemma3ForConditionalGeneration), but I couldn't find and modify it in `modular_gemma3.py`. Where should I apply this fix?